code/modules/mob/living/living.dm

```diff
@@ -703,34 +703,43 @@ Thanks.
 							pulling.Move(T, get_dir(pulling, T))
 							if(M && secondarypull)
 								M.start_pulling(secondarypull)
-							//this is the gay blood on floor shit -- Added back --snx
-							var/turf/TM = M.loc	
-							if (TM.has_gravity() && M.lying && (prob(M.getBruteLoss() / 3)))	
-								if(prob(6)) //Too much bloooooood
+							/* Drag damage is here!*/							
+							if(M.drag_damage() && !(M.health - M.halloss <= config.health_threshold_softcrit))
+								if(prob(M.getBruteLoss() / 5)) //Chance to damage
+									for(var/datum/organ/external/damagedorgan in M.drag_damage())
```

Comment:
I don't understand why that's good or bad, but more importantly, I don't really understand what you're saying I *should* do.

Keep the mob-level drag_damage() proc but make it not return a list of organs, and make drag_damage_human() at the human level that does?